docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3.8"
+
+services:
+    nats:
+        image: nats:latest
+        command: []
+        ports:
+            - "4222:4222"
+            - "6222:6222"
+            - "8222:8222"
+    authorization-service:
+        image: jwalab/authorization-service:latest
+        ports:
+            - "8999:8999"
+    airlock:
+        image: jwalab/airlock:latest
```

Comment:
let's pin this version too, the most recent airlock is 0.0.7

Reply:
Updated. Thank you.